### PR TITLE
Add update to Anyone or PIMW

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/word-email-summaries.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ When users share a Word document in SharePoint or OneDrive, recipients get an em
 - The key points in the doc (derived from content extracted from the document by artificial intelligence)
 - An estimated time to read the doc (based on the word count and the average reading speed in English) 
 
-This summary info helps recipients plan for the work that the doc will require. 
+This summary info helps recipients plan for the work that the doc will require. Currently, this feature only works if the selected links are "Anyone" or "People in my organization".
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/word-email-summaries.md:30]:
```suggestion
This summary info helps recipients plan for the work that the doc will require. Currently, this feature only works if the selected links are "Anyone" or "People in my organization."
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @salarson, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**